Mathlib/Data/Multiset/Lattice.lean

```diff
@@ -10,6 +10,7 @@ Authors: Mario Carneiro
 -/
 import Mathlib.Data.Multiset.FinsetOps
 import Mathlib.Data.Multiset.Fold
+import Mathlib.Data.Multiset.Basic
```

Comment:
Why do you need this import?

Reply:
I thought it was causing the "porting note" issue in l98, but that turned out to be a different problem
removing :)